### PR TITLE
'haiku'

### PR DESCRIPTION
**Pull request by Anandbrewery**
Thanks! for reading my pull request

### CODE REVIEW

**Comment by Anandbrewery** [comment]
hey  please can i know why my pull request is not accepted

File: _haikus/2022hacktoberfest_Anand.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: haiku
+title: 2022 hacktoberfest
+author:Anandbrewery
```
Comment (ltagliaferri) [inline _haikus/2022hacktoberfest_Anand.md:4]: Add a space to format the header correctly

```suggestion
author: Anandbrewery
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thanks for your interest in this repository. We would like contributors to provide an original haiku. Please refer to our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) to learn more.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by Anandbrewery** [comment]
@ltagliaferri  please do check now i have updated

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thanks for your interest in this repository. We would like contributors to provide an original haiku. Please refer to our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) to learn more.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by ltagliaferri** [comment]
Thank you for opening this pull request. The changes requested have not been addressed over the course of a few days, so we are closing this PR due to inactivity and are marking it as `invalid`. If you would like to try again, please open a new pull request. 

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Pull request closed without merging**